sui-execution/src/executor.rs

```diff
@@ -74,6 +74,23 @@ pub trait Executor {
         Result<Vec<ExecutionResult>, ExecutionError>,
     );
 
+    fn commit_transaction_without_execution(
```

Comment:
this function doesn't commit to the store, so we should rename it - perhaps `execute_failed_transaction_to_effects`? 

Alternatively it might be better still to stick with a single function `execute_transaction_to_effects` and extend it to pass in an optional "forced failure" status.

Reply:
this topic has already been addressed in earlier comments